File: java-integration/src/main/java/br/com/acqio/dto/TransactionRequest.java
package br.com.acqio.dto;

import br.com.acqio.enums.CardApplication;
import br.com.acqio.enums.TenantPlanMode;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.math.BigDecimal;
import java.util.ArrayList;
import java.util.List;

@Getter
@Setter
@ToString
public class TransactionRequest {
    private Integer installment = 1;
    private BigDecimal value;
    private CardApplication cardApplication;
    private TenantPlanMode tenantPlanMode = TenantPlanMode.DISABLED;
    private List<BinRange> binsAvailable = new ArrayList<>();
}

File: csharp/csharp/dto/TransactionRequest.cs
using csharp.enums;
using System.Collections.Generic;

namespace csharp.dto
{
    class TransactionRequest
    {
        private int installment = 1;
        private double value;
        private CardApplication cardApplication;
        private TenantPlanMode tenantPlanMode = TenantPlanMode.DISABLED;
        private List<BinRange> binRange = new List<BinRange>();

        public int Installment
        {
            get { return installment; }
            set { installment = value; }
        }

        public double Value
        {
            get { return value; }
            set { this.value = value; }
        }

        public CardApplication CardApplication
        {
            get { return cardApplication; }
            set { cardApplication = value; }
        }

        public TenantPlanMode TenantPlanMode
        {
            get { return tenantPlanMode; }
            set { tenantPlanMode = value; }
        }

        public List<BinRange> BinRange
        {
            get { return binRange; }
            set { binRange = value; }
        }

    }
}